### PR TITLE
Respect time

### PR DESCRIPTION
**Pull request by andreirtaylor**
This is based off of jonahs branch I only made changes to worker/updatechorals

Now the chorals actually respect the timeouts as defined by the user.

### CODE REVIEW

**Comment by jboretsky** [comment]
Also can you remove the get handler here: https://github.com/ChoralCloud/ChoralWeb/blob/master/worker.js#L28-L30

File: worker/updateChorals.js [hunk 1]
```diff
@@ -2,42 +2,138 @@ var client = require('redis').createClient(process.env.REDIS_STORE_URI)
...
+      //
+      // also note that you have to set the timeout here so that the
+      // the choral does not execute in the time between now and when the
+      // first timeout is fired
+      TIMEOUTS[choral.choralId].timeout_id = setTimeout(() => {
+        getChildrenAndRunChoral(choral)
+        TIMEOUTS[choral.choralId].timeout_id = setInterval(() => {
+          getChildrenAndRunChoral(choral)
+        }, choral.sampleRate * 1000)
+      }, run_in)
```
Comment (jboretsky) [inline worker/updateChorals.js:109]: I think I understand this, but why do we need to do it? Couldn't we just start the interval from the beginning?

**Comment by andreirtaylor** [comment]
When there are chorals with long intervals they potentially never will run
if you aren't clever about this.

**Comment by jboretsky** [comment]
Cool, thanks, LGTM :thumbsup: @cpjk

**Pull request merged**